third_party/random_uuid/uuid-swg.js

```diff
@@ -17,8 +17,20 @@ Dual licensed under the MIT and GPL licenses.
  * since we are only using uuidFast().
  */
 
-const CHARS =
-    '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'.split('');
+const CHARS = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'.split(
+  ''
+);
+
+export function getRandomFloat() {
+  try {
+    const arr = new Uint32Array(2);
+    crypto.getRandomValues(arr);
+    const mantissa = arr[0] * Math.pow(2, 20) + (arr[1] >>> 12);
+    return mantissa * Math.pow(2, -52);
+  } catch (e) {
+    return Math.random(); //for older browsers
```

Comment:
What kind of exception are we expecting here? Can we just test for `typeof crypto && crypto.getRandomValues == 'function'`?

Reply:
Makes sense, I just validated crypto && crypto.getRandomValues since it seems unlikely it will be some other random thing that isn't a function.